app/containers/EditProfilePage/renderFileInput.js

```diff
@@ -0,0 +1,106 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import AvatarEditor from 'react-avatar-editor';
+import { FormattedMessage } from 'react-intl';
+
+import {
+  AvatarEditorBorder,
+  AvatarEditorWidth,
+  AvatarEditorColor,
+  AvatarEditorScale,
+  AvatarEditorRotate,
+} from 'containers/Profile/Wrapper';
+
+import messages from 'containers/Profile/messages';
+
+/* eslint-disable */
+function renderFileInput({
+  input,
+  label,
+  sendProps,
+  meta: { touched, error, warning },
+}) {
+  let avatarRefs;
+  let avatarWidth = window.innerWidth > 480 ? AvatarEditorWidth : 320 - 2*AvatarEditorBorder;
```

Comment:
Does it create avatar of different size if used on mobile device or desktop? Avatar should have the same size everywhere. Let's use smaller size that work for both. It is good to define size in constants.

Reply:
reduced size of image